app/pages/order-task-list/contextCreator.js

```diff
@@ -1,23 +1,58 @@
 import neworderPageManifest from './neworder/manifest.json';
 import existingorderPageManifest from './existingorder/manifest.json';
 import taskListManifest from './taskListManifest.json';
+import commonManifest from './commonManifest.json';
 import { generateTaskList } from './helpers/generateTaskList';
 import { baseUrl } from '../../config';
 
 export const getNewOrderContext = ({ orderId }) => ({
+  ...commonManifest,
   ...neworderPageManifest,
-  taskList: generateTaskList({ orderId, taskListManifest }),
   backLinkHref: `${baseUrl}/organisation`,
   orderId,
+  taskList: generateTaskList({ orderId, taskListManifest }),
+  deleteOrderButton: {
+    text: commonManifest.deleteOrderButton.text,
+    altText: commonManifest.deleteOrderButton.disabledAltText,
```

Comment:
if you change `disabledAltText` to match the key you can remove L15 and 16 and do `...commonManifest.deleteOrderButton` and the same for the others

Reply:
Same explanation as above really. It's when I wanted to change these altText values when the button is enabled it started to just get kind of confusing.